R/system.R

```diff
@@ -1439,6 +1452,8 @@ executeGoogleBigQuery <- function(project, query, destinationTable, pageSize = 1
     tb <- bigrquery::bq_project_query(x = project, query = query, quiet = TRUE, use_legacy_sql = !isStandardSQL)
     df <- bigrquery::bq_table_download(x = tb, max_results = Inf, page_size = pageSize, max_connections = max_connections, quiet = TRUE)
   }
+  # Set original scipen
+  options(scipen = original_scipen)
```

Comment:
Make sure this reverting of scipen is done even in error case, using tryCatch.

Reply:
changed it to use tryCatch